service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsLAController.java

```diff
@@ -2,6 +2,7 @@
 
 import io.swagger.annotations.Api;
```

Comment:
Is it only on the LA flow (& not the solicitor) that we can upload SWET?

Reply:
Checked in AAT. Only LA can upload SWET.